games/data/tcg-card-shop-simulator.yml

```diff
@@ -10,48 +10,84 @@ thunderstore:
           label: "Mods"
         modpacks:
           label: "Modpacks"
-        tools:
-          label: "Tools"
-        misc:
-          label: "Misc"
+        apps:
+          label: "Apps"
+        asset-replacement:
+          label: "Asset Replacement"
         audio:
           label: "Audio"
-        sets:
-          label: "Sets"
-        expansions:
-          label: "Expansions"
         cards:
           label: "Cards"
-        npcs:
-          label: "NPCs"
+        decoration:
+          label: "Decoration"
+        expansions:
+          label: "Expansions"
+        furniture:
+          label: "Furniture"
         items:
           label: "Items"
-        asset-replacement:
-          label: "Asset Replacement"
-        qol:
-          label: "QOL"
+        npcs:
+          label: "NPCs"
         phone-related:
           label: "Phone Related"
-        furniture:
-          label: "Furniture"
-        decoration:
-          label: "Decoration"
+        qol:
+          label: "QOL"
+        sellables:
+          label: "Sellables"
+        sets:
+          label: "Sets"
         shop:
           label: "Shop"
-        apps:
-          label: "Apps"
         themes:
           label: "Themes"
+        tools:
+          label: "Tools"
+        misc:
+          label: "Misc"
+        0-62-branch:
+          label: "0.62 Branch"
+        add-entirely-new-cards:
```

Comment:
I know nothing about the mods for this game, but what's the purpose of these "Utilizes X" categories? I'd imagine that if I'm filtering the mods to find interesting ones, I wouldn't be interested if a mod uses e.g. "texture replacer" or "custom texture replacer", I'd probably be using the "asset replacement" category. If this is about package dependencies, it should be handled in package's manifest.json files, not with categories.

Another example, as a user, I would more likely to be using a category "New cards" than something called ""Utilizes Add Entirely New Cards".

I'm asking since removing categories is currently ~impossible, so we'd like to be sure the added categories are actually needed.